pkg/descheduler/descheduler.go

```diff
@@ -404,7 +415,7 @@ func RunDeschedulerStrategies(ctx context.Context, rs *options.DeschedulerServer
 					if pgFnc, exists := pluginsMap[string(name)]; exists {
 						pgFnc(childCtx, nodes, params, handle)
 					} else {
-						f(childCtx, rs.Client, strategy, nodes, podEvictor, evictorFilter, getPodsAssignedToNode)
+						f(childCtx, rs.Client, strategy, nodes, podEvictor, evictorFilter.(framework.EvictorPlugin), getPodsAssignedToNode)
```

Comment:
it's kind of weird to be casting `evictorFilter.(framework.EvictorPlugin)` in the function call. I guess based on the lines above that is only necessary for the migration wrapper. But could `defaultevictor.New()` just return an `EvictorPlugin` instead of a `framework.Plugin`?

Reply:
Your suggestion was what I was doing before: https://github.com/kubernetes-sigs/descheduler/pull/929#discussion_r962644755 :(

Reply:
cc @ingvagabund